tests/main/remove-auto-connections/test-snap-v1/meta/snap.yaml

```diff
@@ -0,0 +1,8 @@
+name: simplesnap
+version: 1.0
+summary: Simple snap
+
+apps:
```

Comment:
You don't seem to need the apps entry at all. You can declare the plug globally.